Sources/Shared/Request+Album.swift

```diff
@@ -29,13 +29,71 @@ public typealias AlbumRequest = Request<Album>
 /// `Request` that returns an array of `Album` objects.
 public typealias AlbumListRequest = Request<[Album]>
 
+fileprivate enum Key {
+    static let name = "name"
+    static let description = "description"
+    static let password = "password"
+    static let privacy = "privacy"
+}
+
 public extension Request {
-    
     /// Returns a new request to fetch a specific album.
     ///
     /// - Parameter uri: The album's URI.
     /// - Returns: Returns a new `Request` for an individual album.
     static func albumRequest(for uri: String) -> Request {
         return Request(path: uri)
     }
+
+    /// Returns a new request for creating an album.
+    /// - Parameter userURI: The URI for the current user, and owner of the album being created.
+    /// - Parameter name: The name of the album being created.
+    /// - Parameter description: An optional description for the album.
+    /// - Parameter privacy: The privacy parameter as a String. If none is specified it defaults to "anyone".
+    /// - Parameter password: An optional password parameter, only required when privacy is set to "password".
+    static func createAlbumRequest(
```

Comment:
I'm not sure how the `Album` model is setup but would it make sense to pass a locally constructed `Album` here instead of its individual attributes? Depending on use case this list of parameters could get pretty long!

Reply:
This is an implementation detail in the app, but the presenter we're working with is only aware of a view-model, and we're relying on the server to return an actual `Album` object if the `.POST` is successful.